packages/interactive_media_ads/lib/src/ad.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2013 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'platform_interface/platform_interface.dart';
+
+/// Data object representing a single ad.
+class Ad {
+  /// Constructs an [Ad] from a specific platform implementation.
+  Ad.fromPlatform(this.platform);
+
+  /// Implementation of [PlatformAd] for the current platform.
+  final PlatformAd platform;
+
+  /// The ad ID as specified in the VAST response.
+  String get adId => platform.adId;
```

Comment:
This is inconsistent with the previous file's capitalization of ID; the Dart API surface should consistently use one or the other. (Same with other names below.)

Reply:
Good catch. Android and iOS use different capitalization so I end up switching on accident. It looks like the precedent is already to use lowercase so I updated the other values.